.gitignore

```diff
@@ -0,0 +1 @@
+*.env
```

Comment:
for the codeship.aes file also add
*.aes

Reply:
Done